RecCalorimeter/src/components/CaloTowerTool.cpp

```diff
@@ -49,57 +52,57 @@ StatusCode CaloTowerTool::initialize() {
-    error() << "Wrong type of segmentation" << endmsg;
-    return StatusCode::FAILURE;
-  }
-  info() << "Retrieving Hcal forward segmentation" << endmsg;
-  tmpPair = retrieveSegmentation(m_hcalFwdReadoutName);
-  m_hcalFwdSegmentation = tmpPair.first;
-  m_hcalFwdSegmentationType = tmpPair.second;
-  if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
-    error() << "Wrong type of segmentation" << endmsg;
-    return StatusCode::FAILURE;
-  }
-
+  //info() << "Retrieving Ecal endcap segmentation" << endmsg;
+  //tmpPair = retrieveSegmentation(m_ecalEndcapReadoutName);
+  //m_ecalEndcapSegmentation = tmpPair.first;
+  //m_ecalEndcapSegmentationType = tmpPair.second;
+  //if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
+  //  error() << "Wrong type of segmentation" << endmsg;
+  //  return StatusCode::FAILURE;
+  //}
+
+  //info() << "Retrieving Ecal forward segmentation" << endmsg;
+  //tmpPair = retrieveSegmentation(m_ecalFwdReadoutName);
+  //m_ecalFwdSegmentation = tmpPair.first;
+  //m_ecalFwdSegmentationType = tmpPair.second;
+  //if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
+  //  error() << "Wrong type of segmentation" << endmsg;
+  //  return StatusCode::FAILURE;
+  //}
+  //info() << "Retrieving Hcal barrel segmentation" << endmsg;
+  //tmpPair = retrieveSegmentation(m_hcalBarrelReadoutName);
+  //m_hcalBarrelSegmentation = tmpPair.first;
+  //m_hcalBarrelSegmentationType = tmpPair.second;
+  //if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
+  //  error() << "Wrong type of segmentation" << endmsg;
+  //  return StatusCode::FAILURE;
+  //}
+  //info() << "Retrieving Hcal extended barrel segmentation" << endmsg;
+  //tmpPair = retrieveSegmentation(m_hcalExtBarrelReadoutName);
+  //m_hcalExtBarrelSegmentation = tmpPair.first;
+  //m_hcalExtBarrelSegmentationType = tmpPair.second;
+  //if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
+  //  error() << "Wrong type of segmentation" << endmsg;
+  //  return StatusCode::FAILURE;
+  //}
+  //info() << "Retrieving Hcal endcap segmentation" << endmsg;
+  //tmpPair = retrieveSegmentation(m_hcalEndcapReadoutName);
+  //m_hcalEndcapSegmentation = tmpPair.first;
+  //m_hcalEndcapSegmentationType = tmpPair.second;
+  //if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
+  //  error() << "Wrong type of segmentation" << endmsg;
+  //  return StatusCode::FAILURE;
+  //}
+  //info() << "Retrieving Hcal forward segmentation" << endmsg;
+  //tmpPair = retrieveSegmentation(m_hcalFwdReadoutName);
+  //m_hcalFwdSegmentation = tmpPair.first;
+  //m_hcalFwdSegmentationType = tmpPair.second;
+  //if (tmpPair.first != nullptr && tmpPair.second == SegmentationType::kWrong) {
+  //  error() << "Wrong type of segmentation" << endmsg;
+  //  return StatusCode::FAILURE;
+  //}
```

Comment:
Why this lines are commented out? Is there a plan to use them in the future?

Reply:
Now I only make that work in ECal Barrel, so the other parts of the detector are commented.
Yes these lines are going to be used in the future.